framework/src/Volo.Abp.AspNetCore.Components.MauiBlazor/Volo/Abp/AspNetCore/Components/MauiBlazor/MauiBlazorRemoteTenantStore.cs

```diff
@@ -57,6 +58,11 @@ public MauiBlazorRemoteTenantStore(AbpTenantClientProxy tenantAppService, IDistr
         return tenantConfiguration;
     }
 
+    public Task<IReadOnlyList<TenantConfiguration>> GetListAsync(bool includeDetails = false)
+    {
```

Comment:
It shouldn't return an empty set, otherwise there's no point in adding this method.

Reply:
That would be a problem on the security side; otherwise, we should add permission check for the user to have tenant list access. Since all application uses the tenant management module [TenantStore](https://github.com/abpframework/abp/blob/8ba6796c06b46d21ff1cff5f54ed07cba607bb55/modules/tenant-management/src/Volo.Abp.TenantManagement.Domain/Volo/Abp/TenantManagement/TenantStore.cs#L11) excepts mvc client and maui blazor (which we don't need this method on client applications).